src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+                    <FavoriteDappsSection
+                      filter={selectedFilter}
+                      removeFilter={() => {
+                        setSelectedFilter({ id: 'all', name: t('dappsScreen.allDapps') })
+                        horizontalScrollView.current?.scrollTo({ x: 0, animated: true })
+                      }}
+                      onPressDapp={onSelectDapp}
+                    />
+                  </>
+                )}
+
+                {dappFavoritesEnabled && (
+                  <Text style={styles.sectionTitle}>{t('dappsScreen.allDappsUpper')}</Text>
+                )}
+              </>
+            }
+            style={styles.sectionList}
+            contentContainerStyle={{
+              padding: Spacing.Thick24,
+              paddingBottom: Math.max(insets.bottom, Spacing.Regular16),
+              flexGrow: 1,
+            }}
+            // Workaround iOS setting an incorrect automatic inset at the top
+            scrollIndicatorInsets={{ top: 0.01 }}
+            scrollEventThrottle={16}
+            onScroll={onScroll}
+            sections={parseResultsIntoAll(dappList, selectedFilter, favoriteDappsById)}
+            renderItem={({ item: dapp }) => (
+              <DappCard
+                dapp={dapp}
+                section={DappSection.All}
+                onPressDapp={onSelectDapp}
+                onFavoriteDapp={onFavoriteDapp}
+              />
+            )}
+            keyExtractor={(dapp: DappV1 | DappV2) => dapp.id}
+            stickySectionHeadersEnabled={false}
+            testID="DAppsExplorerScreen/DappsList"
+            ListEmptyComponent={
+              <NoResults
+                filter={selectedFilter}
+                removeFilter={() => {
+                  setSelectedFilter({ id: 'all', name: t('dappsScreen.allDapps') })
+                  horizontalScrollView.current?.scrollTo({ x: 0, animated: true })
+                }}
+                testID="DAppsExplorerScreen"
+              />
+            }
+            ListFooterComponentStyle={{ flex: 1, justifyContent: 'flex-end' }}
+          />
+        )}
+      </>
+
+      {DappFavoritedToast}
+      {DappInfoBottomSheet}
+    </SafeAreaView>
+  )
+}
+
+function parseResultsIntoAll(
+  dappList: any,
+  filter: DappFilter,
+  favoriteDappsById: string[]
+): SectionData[] {
+  // Prevent favorite dapps from showing up in the all dapps section
+  const data =
+    filter.id === 'all'
+      ? dappList.filter((dapp: DappV2) => !favoriteDappsById.includes(dapp.id))
+      : dappList.filter(
+          (dapp: DappV2) =>
+            !favoriteDappsById.includes(dapp.id) &&
+            dapp.categories &&
+            dapp.categories.includes(filter.id)
+        )
+  // Return empty array if no results
+  if (data.length === 0) return []
+  // Else return dapps in all section
+  return [
+    {
+      data,
+      category: 'all' as unknown as CategoryWithDapps,
```

Comment:
if you follow my comment above and update the type of `SectionData`, you can remove this `as unknown as CategoryWithDapps`

Reply:
Fixed in 5e4e9be0f975baa845bf43846989f61cb2a4ef99!